.github/workflows/build.yml

```diff
@@ -184,16 +184,15 @@ jobs:
           curl -L -o ./bundled/geoip.dat https://github.com/v2rayA/dist-v2ray-rules-dat/raw/master/geoip.dat
           curl -L -o ./bundled/geosite.dat https://github.com/v2rayA/dist-v2ray-rules-dat/raw/master/geosite.dat
 
-      - name: Smoking test
-        if: matrix.goarch == 'amd64' && matrix.goamd64 == 'v1'
-        run: ./bundled/${{ steps.get_filename.outputs.BUNDLE_NAME }} --version
-
       - name: Upload artifact - bundle
         uses: actions/upload-artifact@v3
         with:
           name: ${{ steps.get_filename.outputs.BUNDLE_NAME }}
           path: bundled/*
 
+      - name: Smoking test
+        if: matrix.goarch == 'amd64' && matrix.goamd64 == 'v1'
```

Comment:
`run` was forgotten?

Reply:
Fixed in [06b9470](https://github.com/daeuniverse/daed/pull/42/commits/06b9470e8a93da68473e7e846f62c4f891b1c32a)